### ISSUE TITLE
Redstone Control/Indicator for Machines

### ISSUE THREAD

**Issue by SnowShock35** [ForestryMC/Binnie#377]
Hi,

I have a feature suggestion for Binnie's Mods for Minecraft 1.12.2. I've done my best to describe what the suggestion is below. Should further explanation be required please don't hesitate to ask and I will try my best to explain it more clearly.

In the most recent version of Binnie's mods there is no indication as to whether the machine is being powered by redstone. Forestry, in all of its machines has a tab indicated the machine is being powered by redstone and will no longer function. Since Binnie's mods is an addon to Forestry it only seems right that it would follow this practice, preferably executed in the same manor. 

Tl;Dr; Add a machine sidebar tab thingy when the machine is receiving a redstone signal e.g. like how it is in Forestry.
